src/components/Dialog/Dialog.visual.tsx

```diff
@@ -3,36 +3,46 @@ import { visualize, story, snap } from 'storybook-snapper';
 import { Dialog } from './';
 import { Text, TYPOGRAPHY } from '../Text';
 import { Button, PRIORITY } from '../Button';
+// import { setDarkPalette } from '../../test/visualTestUtils';
+
+const DialogWithContent = props => (
+  <Dialog isOpen wiredToSiteColors={props.wired}>
+    <div className="content" style={{ textAlign: 'center' }}>
+      <Text typography={TYPOGRAPHY.largeTitle}>Are You Sure?</Text>
+      <div
+        className="text-container"
+        style={{ marginTop: '24px', marginBottom: '36px' }}
+      >
+        <Text typography={TYPOGRAPHY.listText} tagName="div">
+          <div>Do you really want to delete the selected files?</div>
+          <div>Once removed, cannot be undone.</div>
+        </Text>
+      </div>
+      <Button
+        upgrade
+        priority={PRIORITY.basicSecondary}
+        style={{ marginLeft: '10px' }}
+      >
+        SECONDARY
+      </Button>
+      <Button upgrade style={{ marginLeft: '10px' }}>
+        PRIMARY
+      </Button>
+    </div>
+  </Dialog>
+);
 
 visualize('Dialog', () => {
   story('simple', () => {
     snap('default props', <Dialog isOpen />);
-    snap(
-      'Dialog with some content',
-      <Dialog isOpen>
-        <div className="content" style={{ textAlign: 'center' }}>
-          <Text typography={TYPOGRAPHY.largeTitle}>Are You Sure?</Text>
-          <div
-            className="text-container"
-            style={{ marginTop: '24px', marginBottom: '36px' }}
-          >
-            <Text typography={TYPOGRAPHY.listText} tagName="div">
-              <div>Do you really want to delete the selected files?</div>
-              <div>Once removed, cannot be undone.</div>
-            </Text>
-          </div>
-          <Button
-            upgrade
-            priority={PRIORITY.basicSecondary}
-            style={{ marginLeft: '10px' }}
-          >
-            SECONDARY
-          </Button>
-          <Button upgrade style={{ marginLeft: '10px' }}>
-            PRIMARY
-          </Button>
-        </div>
-      </Dialog>,
-    );
+    snap('Dialog with some content', <DialogWithContent />);
   });
+
+  // story('Wired to palette', () => {
+  //   snap('Dialog with some content', () => {
+  //     // setDarkPalette();
+  //
+  //     return <DialogWithContent wired />;
+  //   });
+  // });
```

Comment:
Redundant?

Reply:
I wanted to leave this on purpose.
I tried testing the component in a dark palette, but currently changing the palette affects the whole batch of tests, and not only this specific one.
I've opened a feature request for Applitools, I hope they will give us a way to perform a cleanup after a test is run.
This way we can change the palette to the default one after this test is performed

Reply:
So maybe we could add a todo or a comment.